ci/test/00_setup_env_native_valgrind.sh

```diff
@@ -5,11 +5,10 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 export LC_ALL=C.UTF-8
-
+export TEST_RUNNER_EXTRA="--exclude feature_block,rpc_bind,feature_assumevalid,feature_cltv,feature_abortnode,wallet_createwallet,wallet_multiwallet"
```

Comment:
ok, then maybe revert the changes to this file altogether, but I'd still like to get this merged. The timeout changes help me running the valgrind tests on my own ci infrastructure. We can look into how to enable them on travis later on.